### PR TITLE
project weather app

### PR DESCRIPTION
**Pull request by kathinka**
## Netlify link
 Netlify link [here](https://oslo-weather.netlify.app/).

### CODE REVIEW

**Review by lunek1 — COMMENTED** [review]
Hi Kathinka! Just looked at your weather app and code is working fine! As far as I can tell you've checked all the requirements and followed the guidelines. I liked that you added more data, forecast for coming days and what the temperature feels like. It give a really nice touch to the webpage!

Some comments:
- The temperature is rounding to whole numbers if I'm not mistaken. In that case for visual purposes it would be easier on the eyes if you had 0 decimal places.
- There are two small white boxes on each side of the weekdays which I'm guessing isn't suppose to be there. Might be something in CSS that is causing it. 
- Adding a few a comments to your code would help to make it easier to understand its logic and functions and kind of guide you through it upon reviewing it.

Good job on your project and thanks for letting me review it! 🌻

**Comment by kathinka** [comment]
Hi Cornelia! Thanks for reviewing! the task for step two said to display "the temperature (rounded to 1 decimal place)" but I agree - it would look better with only one digit.
I'll look into adding some comments in my code as well. 
as for the styling bug I would like some more info (browser, os and device) as I cant reproduce it, but would like to fix it!

Thanks for all the comments :)

**Comment by lunek1** [comment]
Hi! Sorry for late respond!
 "the temperature (rounded to 1 decimal place)" - I apparently missed this step. It looks so messy (perfectionist here) but okey then you are all good! 🌻
 I also checked in on the css glitch or whatever it could be. I looked at your weather app again but on two other computers (mac and pc) and I couldn't see it. However I can still see it on my laptop. I can't figure out what is causing it so It's probably nothing. Don't worry about it. Good job again! 🙌✨

File: design/.gitignore
Comment (AntonellaMorittu) [inline design/.gitignore:1]: .gitignore should live in the root folder, outside of any others. Please move it outside of the design folder 👍

File: index.html
Comment (AntonellaMorittu) [inline index.html:1]: please format your code using 2tabs spacing 🪄

File: script.js
Comment (AntonellaMorittu) [inline script.js:1]: same here, format the code using 2tabs spacing and improve readability 👍

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hello Kathinka, good job this week! The app is working as it should, you completed almost all the basic requirements to pass this assignment. Please consider this:

- the app should follow as close as possible the chosen design
[this](https://github.com/Technigo/project-weather-app/blob/master/design/design2/iPhone%20X-XS%20%E2%80%93%201.png) was the reference, on mobile your app should look exactly like this.

Please update the UI and you're good to go 🚀 Thank you ⭐

File: design/.gitignore
Comment (kathinka) [reply design/.gitignore:1]: woops, thank you for catching that

File: index.html
Comment (kathinka) [reply index.html:1]: done

File: script.js
Comment (kathinka) [reply script.js:1]: done

**Comment by kathinka** [comment]
All changes have been implemented

**Comment by HIPPIEKICK** [comment]
Consider removing some parts in the mobile view, because it doesn't fit right now:

Skärmavbild 2024-03-12 kl  14 47 58

**Comment by kathinka** [comment]
Aha - what device, os and browser is this? it did not have this issue when
I tested it earlier.  but ok, good suggestion.

tir. 12. mars 2024 kl. 14:49 skrev Matilda Brunemalm <
***@***.***>:

**Comment by kathinka** [comment]
fixed it! (but would still like to know device, os and browser :) )

> Consider removing some parts in the mobile view, because it doesn't fit right now:
> 
> Skärmavbild 2024-03-12 kl 14 47 58

**Review by HIPPIEKICK — APPROVED** [review]
Good job implementing the changes ⭐
For transparency, I used iPhone 5/SE dimenstions in the Chrome inspector, but I have a similar narrow phone (Xperia).

**Comment by kathinka** [comment]
@AntonellaMorittu & @HIPPIEKICK care to take a second look? :)

**Pull request closed without merging**